app/sync.js

```diff
@@ -134,7 +132,7 @@ module.exports.onSyncReady = (isFirstRun, e) => {
   // Sync site settings in case they changed while sync was disabled
   const siteSettings =
     appState.get('siteSettings').filter((value, key) => {
-      return syncUtil.isSyncable('siteSetting', value)
+      return !value.get('objectId') && syncUtil.isSyncable('siteSetting', value)
```

Comment:
not sure this change is correct. ex:
1. a site setting is synced and assigned an object id
2. user disables sync
3. user changes the site setting
4. user re-enables sync

(3) should get synced here